api/everest.go

```diff
@@ -63,10 +63,13 @@ func NewEverestServer(c *config.EverestConfig, l *zap.SugaredLogger) (*EverestSe
 		return nil, errors.New("could not get namespace from Kubernetes")
 	}
 
+	echoServer := echo.New()
+	echoServer.Use(echomiddleware.RateLimiter(echomiddleware.NewRateLimiterMemoryStore(20)))
```

Comment:
20 request per second is used in the echo RateLimiter examples and seems to be a reasonable number. Tried to make usual operations with an Everest deployments - the limit is not exceeded.

Reply:
I think we should make this configurable though. We want everest to be able to support hundreds if not thousands of DBs. At that scale 20 req/s might not be enough.

Reply:
In the latest version I've put the constants to the build flags to increase the limit at least for the debug builds which we use for the API tests. 
@recharte do you think we should make it configurable from the beginning? Should it be an env variable?

Reply:
I think we should. Otherwise, in a few months we'll have customers complaining about their requests being rate limited and we won't be able to help them.